lib/samwise/client.rb

```diff
@@ -33,6 +33,32 @@ def is_excluded?(duns: nil)
       JSON.parse(response.body)["hasKnownExclusion"] == false
     end
 
+    def is_small_business?(duns: nil, naicsCode: nil)
```

Comment:
BTW - this naming comment is related to how predicate matchers work in RSpec:

https://www.relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/predicate-matchers


```ruby
expect(object).to be_small_business

expect(object).to be_is_small_business
```

latter sounds more awkward so I tend to name predicate methods in a way that works well with the matchers. That might not be the best reason I've ever come up with for why I do something, but it's a reason!